.gitignore

```diff
@@ -1 +1,2 @@
 codecept.phar
+node_modules
```

Comment:
The only use case I could think of now is if we want to create tests for our workflows.